react/src/components/awards/SprkAward.js

```diff
@@ -137,10 +162,17 @@ SprkAward.propTypes = {
       src: PropTypes.string.isRequired,
       /** The alt text for the image, icon, or SprkIcon. */
       alt: PropTypes.string.isRequired,
-      /** Additional classes for the image. */
+      /** Deprecated - Use `imageAdditionalClasses` instead.
+       * Additional classes for the image. */
       addClasses: PropTypes.string,
-      /** Additional classes for the link wrapping the image. */
+      /** Deprecated - Use `linkAdditionalClasses` instead.
+       * Additional classes for the link wrapping the image. */
       linkAddClasses: PropTypes.string,
+      /** Additional classes for the image. */
```

Comment:
Hey @bobwise, Lets use a more descriptive comment description for our consumers. We use this one throughout the system for additional classes "A space-separated string of classes to add to the outermost container of the component." We change the part  of " to the outermost container" to something related to the specific use case. So, in this case you would change it to reflect that it is for the link wrapping the image.

Reply:
@afebbraro I'll go ahead and update these, but FYI they don't actually show up in storybook. Nested props like this are only displayed in this (inaccessible) tooltip without the doc comments.

![image](https://user-images.githubusercontent.com/1424560/96466106-bec35a00-11f7-11eb-840a-e4e473b36ecb.png)

Reply:
@bobwise, They only don't show up right now. We want those comments to be consistent so that when we update Storybook (and the feature is added) we then have everything ready to go. Otherwise, we would have to go back and update all these comments.